tests/main/snap-service-stop-mode-sigkill/task.yaml

```diff
@@ -29,6 +33,7 @@ execute: |
 
     echo "When it is re-installed one process uses sigterm, the other sigterm-all"
     install_local test-snapd-service
+    while ! test -f /var/snap/test-snapd-service/common/ready; do sleep 1; done
```

Comment:
Unless I've missed it, this test is missing the cleanup of the `ready` file that the other tests do.

On the other hand, you could also change the `test -f` for a `rm -v`: I mean `while ! rm -v /..../ready; do sleep 1; done` should work just as well and require no cleanup :-)